packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -56,6 +57,10 @@ export default class PciInstancesAddController {
     this.instanceReadMoreUrl =
       INSTANCE_READ_MORE_GUIDE.ALL_GUIDE[this.user.ovhSubsidiary] ||
       INSTANCE_READ_MORE_GUIDE.ALL_GUIDE.DEFAULT;
+    this.windowsPrivateModeLicenseGuideUrl =
+      WINDOWS_PRIVATE_MODE_LICENSE_GUIDE[this.user.language] ||
```

Comment:
Usually we only use the subsidiary and not the language. You also need to modify the constants file.

Reply:
How do you manage url like fr_CA, en_CA, en_GB, en_US, en_AU if you use sub ?

Reply:
In constant file: 
https://github.com/ovh/manager/pull/9421#discussion_r1223925011

Reply:
Done